lib/discovery_api_web/controllers/dataset_query_controller.ex

```diff
@@ -41,14 +41,30 @@ defmodule DiscoveryApiWeb.DatasetQueryController do
     end
   end
 
+  def query(conn, params, "geojson" = format) do
```

Comment:
It would be really, really nice if you could time-box drying up the three `query` functions (or at least this one and `json` one), but it's not a show stopper.

Reply:
One way to do that with a guard clause is
```suggestion
  def query(conn, params, format) when format in ["json", "geojson"] do
```

Reply:
Doesn't refactoring the query functions make the guard clause irrelevant?

Reply:
Yeah that helps